src/graphql/schema.ts

```diff
@@ -61,6 +62,7 @@ type Mutation {
 
 type User {
   avatarUrl: String!
+  backgroundUrl: String!
```

Comment:
no, `avatarUrl` already exists for all users. When the user didn't set it it will be just an empty string. In the case of `backgroundUrl` you are adding a brand new field and none of the users has it.